taskcluster/glean_taskgraph/transforms/module_build.py

```diff
@@ -19,8 +19,9 @@ def build_task(config, tasks):
         name = module_info["name"]
         version = get_version()
 
-        for i,item in enumerate(task["run"]["gradlew"]):
-            task["run"]["gradlew"][i] = task["run"]["gradlew"][i].format(module_name=name)
+        if "gradlew" in task["run"]:
```

Comment:
Once we remove the above `transform` in kind.yaml we won't require these changes. 
The module_build transform doesn't apply here and is Gradle-specific.